Add 2 storyboards on storylines work (#78)
eucp-project/storyboards: static/stories/storylines.md

:::

:::Chapter{headline="Findings 1" image=slide5.png}
## What are climate storylines and where are they useful?

The key points on the left were largely drawn from the two examples of
end-to-end production for storylines for applications.

Storylines as a user product, as well as a scientific process, are highly
dependant on the user or science question, and so we do not attempt to define
what they may be comprised of. However, both the application examples as well as
the emerging science have demonstrated their usefulness.

:::

:::Chapter{headline="Findings 2" image=slide6.png}
## How could storylines bring together various output and products of EUCP science?

The key points on the left were largely drawn from work across EUCP which was
not included in the storylines task. However, the examples of storylines for
applications and scientific building blocks helped speculate how these elements
could come together in the future.

In EUCP this work was performed in parallel. Now these examples and techniques
have been developed, we can begin to think about how storylines, or related
concepts, may form part of the climate data value chain in the future. Informing

> the production of information through to it's application in decision making and

Reviewer: ```suggestion
the production of information through its application in decision making and
```

Is this what you mean?
